server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -142,6 +143,20 @@ func (s *BuildBuddyServer) DeleteInvocation(ctx context.Context, req *inpb.Delet
 	return &inpb.DeleteInvocationResponse{}, nil
 }
 
+func (s *BuildBuddyServer) CancelInvocation(ctx context.Context, req *inpb.CancelInvocationRequest) (*inpb.CancelInvocationResponse, error) {
+	err := s.authorizeInvocationWrite(ctx, req.InvocationId)
```

Comment:
it's a good habit to use the getter (i.e. GetInvocationId) methods on protos as they have additional nil checking.
For example req.GetFoo().GetBar() will not panic even if foo is nil whereas req.Foo.Bar will. 
It's a top level field here so technically it doesn't matter, but it will help avoid issues in more complex code.

Reply:
Done

Reply:
I see you updated the use below but not the one on this line.

Reply:
Oops done